hw1/task2/bash.py

```diff
@@ -0,0 +1,70 @@
+import os
+
+
+def wc(first: str, *args: str):
+    def wc_for_one(filepath: str):
+        size = os.path.getsize(filepath)
+        f = open(filepath)
+        words = 0
+        lines = 0
+        for line in f:
+            lines += 1
+            words += len(line.split())
+        f.close()
+        return {"newlines": lines, "words": words, "bytes": size}
+
+    acc = {"newlines": 0, "words": 0, "bytes": 0}
+    files_not_found = []
+    for filename in [first, *args]:
+        try:
+            file_wc = wc_for_one(filename)
+        except FileNotFoundError:
+            files_not_found.append(filename)
+            continue
+        print(str(file_wc) + "\t" + filename)
```

Comment:
`str(dictionary)` looks very awkward, let's write it nicer, by separating the values with `\t` so that they are also underneath one another. Since you do this in two separate places, you just write another small function for this.